.github/workflows/tests.yml

```diff
@@ -40,18 +56,44 @@ jobs:
         id: set-up-homebrew
         uses: Homebrew/actions/setup-homebrew@master
 
-      - run: brew test-bot --only-tap-syntax
-
       - run: brew test-bot --only-formulae-detect
-        if: github.event_name == 'pull_request'
         id: formulae-detect
 
+      - name: Fetch detected formulae bottles
```

Comment:
@MikeMcQuaid I've added a test to make sure we can fetch the just-built bottles before merging. We can do something like
```yaml
if: github.event_name == 'merge_group'
```
so it's run just before an attempt to merge.

And, yes, don't worry, I'll look at moving this to `brew`/`test-bot`/etc. too. Just writing it down here as a quick POC.

Reply:
Homebrew/homebrew-test-bot#902

Reply:
@carlocab That's really great, thanks!